src/components/Card/Card.js

```diff
@@ -1,31 +1,44 @@
-import React from 'react';
-import "./Card.css"
-const Card = ({id,title,description,date,name,age}) => {
-    return (
-        <div className="post__container">
-            <div className="post">
-                <div className="post-content">
-                    <strong>{id}</strong>
-                    <strong>{title}</strong>
-                    <div>
-                        {description}
-                    </div>
-                    <div>
-                        {date}
-                    </div>
-                    <div>
-                        {name}
-                    </div>
-                    <div>
-                        {age}
-                    </div>
-                </div>
-                <div className="post__buttons">
-                    <button>Удалить</button>
-                </div>
-            </div>
-        </div>
-    );
+import React from "react";
+import "./Card.css";
+
+const Card = ({
+  id,
+  index,
+  title,
+  description,
+  transmission,
+  fuel,
+  volume,
+  mileage,
+  image,
+  date,
+  price,
+}) => {
+  return (
+    <div className="card__container">
+      <div className="card__image">
+        <img src={image} alt="#" />
+      </div>
+      <div className="card__info">
+        <strong>{index + 1}.</strong>
+        <strong>{title}</strong>
+        <div>{description}</div>
+        <div>{transmission}</div>
+        <div>{volume}</div>
+        <div>{fuel}</div>
+        <div>{mileage}</div>
+        <div>{date}</div>
```

Comment:
I think it is not a best solution to use `<div>` for each single property. 
Please read about semantic html elements and we will discuss better solution for that.